src/plugins/media/nodeviews/mediaNodeView/media.tsx

```diff
@@ -50,15 +49,17 @@ export interface MediaNodeProps extends ReactNodeProps, ImageLoaderProps {
 }
 
 interface MediaNodeState {
-  finalUrl?: string
   viewMediaClientConfig?: MediaClientConfig
   contextIdentifierProvider?: ContextIdentifierProvider
 }
 
 export class MediaNode extends Component<MediaNodeProps, MediaNodeState> {
   private mediaPluginState: MediaPluginState
 
-  state: MediaNodeState = {}
+  state: MediaNodeState = {
+    image: '',
```

Comment:
Maybe this should be named imageUrl as it doesn't store image binary but its URL

Reply:
Updated